Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM gcc:latest as build
```

Comment:
Just noticed: this should be a number, not `latest`, otherwise it'll break on its own down the line. Can be just the major semver component though, to get a bit of updates automatically.

Reply:
yep, good catch. I meant to do this